slack-infra/README.md

```diff
@@ -12,6 +12,47 @@ Secrets are stored as `{app}/{secret-name}-secret.yaml` and encrypted with git-c
 
 None of the resources have a namespace defined
 
-## TODO: how to deploy
+## How to deploy
+
+```bash
+# Basic resources
+    kubectl apply -n slack-infra -f ingress.yaml
+    kubectl apply -n slack-infra -f certificate.yaml
+
+    # slack-event-log
+    kubectl apply -n slack-infra -f slack-event-log/deployment.yaml
+    kubectl apply -n slack-infra -f slack-event-log/service.yaml
+
+    # slack-moderator
+    kubectl apply -n slack-infra -f slack-moderator/deployment.yaml
+    kubectl apply -n slack-infra -f slack-moderator/service.yaml
+
+    # slack-welcomer
+    kubectl apply -n slack-infra -f slack-welcomer/deployment.yaml
+    kubectl apply -n slack-infra -f slack-welcomer/message.yaml
+    kubectl apply -n slack-infra -f slack-welcomer/service.yaml
+
+    # slackin
+    kubectl apply -n slack-infra -f slackin/deployment.yaml
+    kubectl apply -n slack-infra -f slackin/service.yaml
```

Comment:
```for f in ./*/*.yaml; do echo $f | grep -q secret || kubectl apply -n slack-infra -f $f; done```

Reply:
I wanted to be more explicit and treat it as some kind of documentation too easily see what are gonna to deploy exactly, but I don't have strong opinion about it. I'm gonna update the PR